metrics/codahale.go

```diff
@@ -291,10 +291,15 @@ func (sm skipperMetrics) MarshalJSON() ([]byte, error) {
 	for name, metric := range sm {
 		values := make(map[string]interface{})
 		var metricsFamily string
+
 		switch m := metric.(type) {
 		case metrics.Gauge:
 			metricsFamily = "gauges"
 			values["value"] = m.Value()
+		case metrics.GaugeFloat64:
```

Comment:
Does it make sense to also add a test similar to this https://github.com/zalando/skipper/blob/master/metrics/prometheus_test.go#L695?

Reply:
It’s not needed as long as you implemented the interface and you did, so it is fine.
Thanks